app/controllers/works_controller.rb

```diff
@@ -17,8 +17,6 @@ def new
 
   def show
     @work = Work.find(params[:id])
-    @datacite = Datacite::Resource.new_from_json(@work.data_cite)
```

Comment:
I also changed the references to the standalone `@datacite` to `@work.datacite_resource` in several files associated with the Work data entry form. Strictly speaking that is not part of the title validation but it was easy to do and makes the code less dependent on an extra variable since we are now using a property in the `@work` object.

Reply:
👍🏻